### PR TITLE
wip :DefaultArgParser.py: Make "--find-config" default

### PR DESCRIPTION
**Pull request by vreethesh**
### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference).
- [ ] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

Removed blocks of code checking if config is default.
Since config is true by default.

Closes https://github.com/coala/coala/issues/2535

### CODE REVIEW

File: coalib/settings/ConfigurationGathering.py [hunk 1]
```diff
@@ -136,9 +136,7 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     cli_sections = parse_cli(arg_list=arg_list, arg_parser=arg_parser)
     check_conflicts(cli_sections)
 
-    if (
-            bool(cli_sections["default"].get("find_config", "False")) and
-            str(cli_sections["default"].get("config")) == ""):
+    if (str(cli_sections["default"].get("config")) == ""):
```
Comment (Nosferatul) [inline coalib/settings/ConfigurationGathering.py]: Your tests failed, please check what CI told you. It seems that `ConfigurationGatheringTest.test_default_coafile_parsing` failed.

**Comment by sils** [comment]
@vreethesh please fix your tests. Ask on gitter if you need help.

**Comment by sils** [comment]
tests still failing, please add `wip: ` to your PR title as long as you aren't sure if tests are passing, ok?

**Comment by jayvdb** [comment]
Continues at https://github.com/coala/coala/issues/2535

**Pull request closed without merging**